### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by BlackDex**
Updated the issue template a bit regarding some remarks in the previous pr.

Also made it so that collapsing all items will show all the specific item id's instead of there types. Easy for editiing :).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -25,71 +30,18 @@ body:
...
-      description: What version of the operating system(s) are you seeing the problem on?
-      placeholder: ex. Arch Linux, Ubuntu 24.04, Windows 11
-  - type: dropdown
-    id: deployment
+  #
+  - id: deployment
+    type: dropdown
     attributes:
       label: Deployment method
       description: How did you deployed Vaultwarden?
```
Comment (stefan0xC) [inline .github/ISSUE_TEMPLATE/bug_report.yml:47]:
```suggestion
      description: How did you deploy Vaultwarden?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -103,21 +55,47 @@ body:
...
+      description: Are you using a reverse proxy, if so which and what version?
+      placeholder: ex. nginx 1.26.2, caddy 2.8.4, traefik 3.1.2, haproxy 3.0
+    validations:
+      required: true
+  #
+  - id: os
+    type: dropdown
+    attributes:
+      label: Host/Server Operating System
+      description: On what operating system are running the Vaultwarden server?
```
Comment (stefan0xC) [inline .github/ISSUE_TEMPLATE/bug_report.yml:78]:
```suggestion
      description: On what operating system are you running the Vaultwarden server?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -131,9 +109,59 @@ body:
...
+      label: Actual Result
+      description: A clear and concise description of what is happening.
+    validations:
+      required: true
+  #
+  - id: logs
+    type: textarea
+    attributes:
+      label: Logs
+      description: Provide the logs generated by Vaultwarden generated during the time this issue occurs.
```
Comment (stefan0xC) [inline .github/ISSUE_TEMPLATE/bug_report.yml:154]:
```suggestion
      description: Provide the logs generated by Vaultwarden during the time this issue occurs.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -2,19 +2,24 @@ name: Bug Report
...
         The [GitHub Discussions](https://github.com/dani-garcia/vaultwarden/discussions/) has sections for Questions and Ideas.
 
-        Also, make sure you are running the [Latest Release](https://github.com/dani-garcia/vaultwarden/releases) of Vaultwarden!
+        Also, make sure you are running [![GitHub Release](https://img.shields.io/github/release/dani-garcia/vaultwarden.svg)](https://github.com/dani-garcia/vaultwarden/releases/latest) of Vaultwarden!
         And search for existing open or closed issues or discussions regarding your topic before posting.
-  - type: textarea
-    id: support-string
+
+        Be sure to check and validate the Vaultwarden Admin Diagnostics (`/admin/diagnostics`) page for any errors!
+        See here how to [Enabling admin page](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page).
```
Comment (stefan0xC) [inline .github/ISSUE_TEMPLATE/bug_report.yml:19]:
```suggestion
        See here [how to enable the admin page](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page).
```

**Review by stefan0xC — CHANGES_REQUESTED** [review]

**Comment by BlackDex** [comment]
Thanks @stefan0xC, i adjusted the items :).

**Review by stefan0xC — APPROVED** [review]

**Review by dani-garcia — APPROVED** [review]

**Pull request merged**